internal/cli/cmd/delete/credentials.go

```diff
@@ -0,0 +1,124 @@
+package delete
+
+import (
+	"context"
+	goerrors "errors"
+	"fmt"
+	"slices"
+
+	"connectrpc.com/connect"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/akuity/kargo/internal/cli/client"
+	"github.com/akuity/kargo/internal/cli/config"
+	"github.com/akuity/kargo/internal/cli/option"
+	v1alpha1 "github.com/akuity/kargo/pkg/api/service/v1alpha1"
+)
+
+type deleteCredentialsOptions struct {
+	*option.Option
+	Config config.CLIConfig
+
+	Names []string
+}
+
+func newDeleteCredentialsCommand(
+	cfg config.CLIConfig,
+	opt *option.Option,
+) *cobra.Command {
+	cmdOpts := &deleteCredentialsOptions{
+		Option: opt,
+		Config: cfg,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "credentials [--project=project] [NAME ...]",
```

Comment:
```suggestion
		Use:     "credentials [--project=project] (NAME ...)",
```

(We require _at least_ one)

Reply:
I probably copied that from "get" and overlooked. Thanks again for the eagle eye!